dotcom-rendering/src/palette.ts

```diff
@@ -3384,8 +3384,73 @@ const pullQuoteIconDark: PaletteFunction = (format: ArticleFormat) => {
 		: text;
 };
 
-const shareIconFillLight: PaletteFunction = ({ design, theme, display }) => {
+const shareButtonLiveBlogMobileLight: PaletteFunction = ({ design, theme }) => {
```

Comment:
I think we discussed this with @georgeblahblah and @JamieB-gu before, but a `PaletteFunction` handles every single variation of `ArticleFormat`, so it should probably be named `shareButtonMobileLight` instead.